Buildings/Applications/DHC/EnergyTransferStations/Examples/CoolingIndirectClosedBuildingLoop.mo

```diff
@@ -0,0 +1,194 @@
+within Buildings.Applications.DHC.EnergyTransferStations.Examples;
+model CoolingIndirectClosedBuildingLoop
+  "Example model for indirect cooling energy transfer station that has a closed chilled water loop on the building side"
+  extends Modelica.Icons.Example;
+
+  package Medium = Buildings.Media.Water;
+
+  parameter Modelica.SIunits.MassFlowRate mDis_flow_nominal = 0.5
+    "Nominal mass flow rate on district-side (primary)";
+  parameter Modelica.SIunits.MassFlowRate mBui_flow_nominal = 0.5
+    "Nominal mass flow rate on building-side (secondary)";
+
+  parameter Modelica.SIunits.SpecificHeatCapacity cp=
+    Medium.specificHeatCapacityCp(
+    Medium.setState_pTX(Medium.p_default, Medium.T_default, Medium.X_default))
+    "Specific heat capacity of medium 1";
+
+  Modelica.Blocks.Sources.Constant TSetCHWS(k=273 + 7)
+    "Setpoint temperature for building chilled water supply"
+    annotation (Placement(transformation(extent={{-120,10},{-100,30}})));
+  Buildings.Fluid.Sources.Boundary_pT sinDis(
+    redeclare package Medium = Medium,
+    T=287.15,
+    nPorts=1) "District (primary) sink" annotation (Placement(transformation(
+        extent={{10,-10},{-10,10}},
+        rotation=90,
+        origin={80,80})));
+  Buildings.Applications.DHC.EnergyTransferStations.CoolingIndirect coo(
+    redeclare package Medium1 = Medium,
+    redeclare package Medium2 = Medium,
```

Comment:
better to redeclare the medium package in one place, which is when instantiated it. so, change it to 
```
redeclare packae Medium = Medium,

```

Reply:
Done for `CoolingIndirectClosedBuildingLoop.mo` and `CoolingIndirectOpenLoops.mo`.